### DISCUSSION TITLE
Read this first!

### DISCUSSION THREAD

**Discussion by khaeru** [iiasa/message_ix#436] [General]
The MESSAGEix development team within the IIASA Energy, Climate, and Environment (ECE) Program provides this space for users of MESSAGEix to exchange advice on using the software.

Please note that:
1. As part of the documentation, we have a carefully-written page listing [**pre-requisite knowledge and skills**](https://docs.messageix.org/en/latest/prereqs.html). If you have not learned this required knowledge and acquired these skills, then it is very likely that you will have difficulty using MESSAGEix.

   We occasionally provide specific learning opportunities for research collaborators and members of select partner groups; please inquire by e-mail if interested. However, in general **we cannot provide free training on demand** for people who lack this knowledge.
2. The Apache License 2.0 under which MESSAGEix is published includes a [**no warranty clause**](https://github.com/iiasa/message_ix/blob/9efac12/LICENSE#L143-L151) that reads (in part):
   > Licensor provides the Work an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND including any warranties or conditions of FITNESS FOR A PARTICULAR PURPOSE. You are solely responsible for determining the appropriateness of using the Work.
  
   In other words, it is up to each user to understand the features of MESSAGEix and decide whether it is suitable for their intended use. The development team is **not obliged to** (a) help users understand the existing features, or (b) provide new features desired by a user.
  
3. **Bugs** in the software—specific cases where the software does not do what the documentation claims—should not be reported here. Instead, [open a new issue](https://github.com/iiasa/message_ix/issues/new).

Funders of IIASA, the ECE Program, or its specific projects and researchers are provided with higher levels of support, on a case-by-case basis.
